test/unittests/NDB_BVL_FeedbackTest.php

```diff
@@ -61,7 +60,7 @@ protected function setUp(): void
      */
     public function testCreateFeedbackTypeWithInvalidName()
     {
-        $this->setExpectedException('LorisException');
+        $this->expectException('LorisException');
```

Comment:
I don't know why this test was passing since setExpectedException was removed from phpunit, but phan picked it up here..